patches/android_webview-BUILD.gn.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/android_webview/BUILD.gn b/android_webview/BUILD.gn
+index 9b758420dc4cf53c998274592e6e4d018280e1de..691af9b3a6842b19266b8535bd4af86f46cd5655 100644
+--- a/android_webview/BUILD.gn
++++ b/android_webview/BUILD.gn
+@@ -706,6 +706,9 @@ source_set("common") {
+     "//android_webview/common",
+     "//android_webview/common:common_mojom",
+     "//base/third_party/dynamic_annotations:dynamic_annotations",
++    "//brave/browser/net",
++    "//brave/browser",
++    "//components/translate/core/language_detection",
+     "//components/about_ui",
```

Comment:
make a patch one string only probably, so we patch less.

Reply:
@SergeyZhukovsky , do you mean to make this way
```
+ +"//brave/browser/net","//brave/browser","//components/translate/core/language_detection", 
``` 
?

Reply:
yep, the less lines we patch the easier to rebase after that. We don't care about how does it look like as it's a patch in any case.

Reply:
actually what we should do here is create a new target in brave called `//brave/android_webview:common` that includes all 3 of these. That way the patch is extensible and more deps can be added in the future without changing the patch.

Reply:
@bridiver @SergeyZhukovsky thanks, minimized patch according to your proposition

Reply:
I think `android_webview` target should not depend on brave layer. It looks like layer violation. WDYT?

Reply:
@jumde is working on this. https://github.com/brave/brave-core/pull/3415#discussion_r333359050

Reply:
@simonhong , yes this had confused me , but I could not find any better option.

Reply:
... and I run into `assert_no_deps = [ "//chrome/*" ]` at https://cs.chromium.org/chromium/src/components/BUILD.gn?l=619 when tried to add deps in obvious way

Reply:
Right, I think this link error should be fixed from `safe_browsing_network_context.cc`